fabfile/setup.py

```diff
@@ -30,6 +31,7 @@ def basic_testing():
     execute(add_workers)
 
 @task
+@roles('master')
```

Comment:
@SaitTalhaNisanci can you add this to `def enterprise():` as well? I think even for `tpch ` and `valgrind ` we should do similar things, right?

Reply:
@onderkalaci `enterprise` is not called with `execute`, rather it is directly called, so in that case this is the expected behavior. For `valgrind` there is a sudo command, I wasnt 100% sure if sudo is executed once per host or only in the localhost. For `tpch` we already have it. I am not sure why `enterprise` is not called with `execute` whereas the others are called with `execute`. So there seems to be some inconsistency, I will see if we can simplify this, ideally we can call the functions with execute, and inside functions  we can call everything as a single call not with `execute`, I am guessing this way everything will be run once as we want to. I will also take a look at this.

Reply:
I wish we can fix enterprise piece at least, that's annoying :/ @SaitTalhaNisanci